### DISCUSSION TITLE
Name the queen with the most lip syncs on ‘RuPaul’s Drag Race’

### DISCUSSION THREAD

**Discussion by PavelLaptev** [ParanoidUser/yolo#205] [Galaxy Trivia] [answered]
And don't f*ck it up…

**Comment by ParanoidUser** [comment, marked as answer]
Thank you for your active participation in trivia. Keep the questions coming! 😊

 I suppose it could be Trinity?

**Comment by PavelLaptev** [comment]
Sure thing :-) You right @ParanoidUser it was Trinity
